client/components/Querries.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
Minor. There is a typo in the file name. It should read `Queries.tsx`. You can use a `git mv <old name> <new name> to preserve the git history for this file when renaming.

Reply:
Ok!